Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/OperationHub.hs

```diff
@@ -70,8 +72,10 @@ getOperationGetRequests (mbPersonId, _, _) mbFrom mbTo mbLimit mbOffset mbStatus
       defaultFrom = UTCTime (utctDay now) 0
       from = fromMaybe defaultFrom mbFrom
       to = fromMaybe now mbTo
-  requests <- QOHRE.findAllRequestsInRange from to limit offset Nothing mbStatus mbType (Just driverId.getId) Nothing (Just rcNo)
-  pure (OperationHubRequestsResp (map fst requests))
+  requests <- QOHRE.findAllRequestsInRange from to limit offset Nothing mbStatus mbType (Just driverId.getId) Nothing Nothing (Just rcNo)
+  pure (OperationHubRequestsResp (map getFirstReq requests))
```

Comment:
looks inefficient, you used join query, but only first table `operationHubRequests` really required, second and third did not used in filters